.travis.yml

```diff
@@ -89,7 +90,9 @@ script:
     - sudo touch /tmp/file_to_overwrite_for_easybuild_test_copy_file.txt
     - sudo chmod o+w /tmp/file_to_overwrite_for_easybuild_test_copy_file.txt
     # run test suite
-    - python -O -m test.framework.suite 2>&1 | tee test_framework_suite.log
+    - coverage run -m test.framework.suite
+    # ignore errors, since tests inject easyblocks for which there are no source files anymore after the tests have run
+    - coverage report -m --ignore-errors
```

Comment:
Please change to:

```
- if [ "x$TRAVIS_PYTHON_VERSION" == 'x2.6' ];
  # coveralls doesn't support Python 2.6 anymore, so only run tests
  then python -O -m test.framework.suite 2>&1 | tee test_framework_suite.log
  # ignore errors, since tests inject easyblocks for which there are no source files anymore after the tests have run
  else coverage run -m test.framework.suite 2>&1 | tee test_framework_suite.log; coverage report -m --ignore-errors;
  fi